web-ng/src/app/components/form-field-editor/form-field-editor.component.css

```diff
@@ -0,0 +1,44 @@
+.field-type {
+    margin-left: 30px;
+    font-size: 12px;
+  }
+  
+  .icon-field-type {
+    vertical-align: middle;
+    margin-right: 20px;
+  }
+  
+  .question-actions {
+    margin-bottom: 20px;
+    margin-right: 10px;
+  }
+  
+  .trash-icon {
+    color: #707074;
```

Comment:
Let's use Angular Material themes instead of specifying specific colors wherever possible. In this case, the mat-button attribute would cause the trash icon to use the default foreground color.